src/fdk.ts

```diff
@@ -14,10 +14,15 @@ import { initializeLogger } from './lib/Logger';
 import { isAThemeDirectory } from './helper/utils';
 import inquirer from 'inquirer';
 import path from 'path';
+import Env from './lib/Env';
+import { getActiveContext } from './helper/utils.js';
+
 const packageJSON = require('../package.json');
 
 const notRequireAuthCommands = ['login', 'env', 'logout', 'auth'];
 const notRequireEnvCommands = ['env'];
+const requiredThemeCommands = ['serve', 'sync', 'new', 'init'];
```

Comment:
define all theme commands as an array in constant and populate use it here @Shivraj0

Reply:
Done @vivek8690